sdk/sdk/OBAApplication.h

```diff
@@ -0,0 +1,45 @@
+//
+//  OBAApplication.h
+//  OneBusAwaySDK
+//
+//  Created by Dima Belov on 4/25/15.
+//  Copyright (c) 2015 One Bus Away. All rights reserved.
```

Comment:
Legal nit - this should be `Copyright (c) 2015 Dima Belov`, and should include the Apache 2.0 license header.  You own the copyright because you created the code, and then you license your work via Apache, and to the OneBusAway project via the ICLA (via UW for now, since we don't yet have a legal "OneBusAway" entity).  Same applies to any other similar files.